docs/src/lib/binary_functions.md

```diff
@@ -20,8 +20,10 @@ cartesian_product(::VPolytope, ::VPolytope)
 
 ## Check for emptiness of intersection
 
+!!! note
+    Starting with Julia v1.5, `isdisjoint` can be used as an alternative name to `is_intersection_empty`.
```

Comment:
We always had this alias.

Reply:
ok. i've edited the line. in this PR i removed the docstrings to circumvent a problem between julia versions -- the problem about "missing docstring" and our hacky workaround , and because our development time is limited  :)

Reply:
(we can always put back the docstring once we do https://github.com/JuliaReach/LazySets.jl/pull/2502 and remove support for v1.0)